settings/development/PandoraSettings_Streaming_Neutrino_DUNEFD.xml

```diff
@@ -0,0 +1,640 @@
+                <algorithm type = "LArSimpleClusterCreation" description = "ClusterRebuilding"/>
+            </algorithm>
+            <algorithm type = "LArHitWidthClusterMerging"/>
+        </AlgorithmsShowerClustersV>
+    </algorithm>
+
+    <algorithm type = "LArClusteringParent">
+        <algorithm type = "LArTrackClusterCreation" description = "ClusterFormation"/>
+        <InputCaloHitListName>CaloHitListW</InputCaloHitListName>
+        <ClusterListName>ClustersW</ClusterListName>
+        <ReplaceCurrentCaloHitList>true</ReplaceCurrentCaloHitList>
+        <ReplaceCurrentClusterList>true</ReplaceCurrentClusterList>
+    </algorithm>
+    <algorithm type = "LArLayerSplitting"/>
+    <algorithm type = "LArDLTrackShowerStreamSelection">
+        <TrackListName>TrackClustersW</TrackListName>
+        <ShowerListName>ShowerClustersW</ShowerListName>
+    </algorithm>
+
+    <algorithm type = "LArStreaming">
+        <InputListNames>TrackClustersW ShowerClustersW</InputListNames>
+        <OutputListNames>TrackClustersW ShowerClustersW</OutputListNames>
+        <AlgorithmsTrackClustersW>
+            <algorithm type = "LArLongitudinalAssociation" />
+            <algorithm type = "LArTransverseAssociation" />
+            <algorithm type = "LArLongitudinalExtension" />
+            <algorithm type = "LArTransverseExtension" />
+            <algorithm type = "LArCrossGapsAssociation" />
+            <algorithm type = "LArCrossGapsExtension" />
+            <algorithm type = "LArOvershootSplitting" />
+            <algorithm type = "LArBranchSplitting" />
+            <algorithm type = "LArKinkSplitting" />
+            <algorithm type = "LArTrackConsolidation">
+                <algorithm type = "LArSimpleClusterCreation" description = "ClusterRebuilding"/>
+            </algorithm>
+            <algorithm type = "LArHitWidthClusterMerging"/>
+        </AlgorithmsTrackClustersW>
+        <AlgorithmsShowerClustersW>
+            <algorithm type = "LArLongitudinalAssociation" />
+            <algorithm type = "LArTransverseAssociation" />
+            <algorithm type = "LArLongitudinalExtension" />
+            <algorithm type = "LArTransverseExtension" />
+            <algorithm type = "LArCrossGapsAssociation" />
+            <algorithm type = "LArCrossGapsExtension" />
+            <algorithm type = "LArOvershootSplitting" />
+            <algorithm type = "LArBranchSplitting" />
+            <algorithm type = "LArKinkSplitting" />
+            <algorithm type = "LArTrackConsolidation">
+                <algorithm type = "LArSimpleClusterCreation" description = "ClusterRebuilding"/>
+            </algorithm>
+            <algorithm type = "LArHitWidthClusterMerging"/>
+        </AlgorithmsShowerClustersW>
+    </algorithm>
+
+    <algorithm type = "LArListDeletion">
+        <ClusterListNames>ClustersU ClustersV ClustersW</ClusterListNames>
+        <VertexListNames>CandidateVertices3D NeutrinoVertices3D</VertexListNames>
```

Comment:
Do these vertices exist at this point? What might be in these cluster lists? Only failures, or expect to see something?

Reply:
There shouldn't be anything,, just being paranoid.